src/components/SearchBar/index.js

```diff
@@ -46,6 +46,7 @@ const SearchBar = ({intl, onFormSubmit}) => {
 
     return (
         <div className='search-bar'>
+            <h4><FormattedMessage id='pick-time-range' /></h4>
```

Comment:
Headings must always be in order: h1, h2, h3, h4,... in any page. Search page would now skip from h1 to h4. Heading sizes can be changes with css.

Reply:
Fixed this, and also the warning about a fieldset missing the legend